### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by chadeddington**
### Changes
- Added an id to album request for testing purposes
- Web Component tester to bower.json
- Added sample data to window object for testing purposes
- Added unit tests

@jshcrowthe

### CODE REVIEW

File: demo/data.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[
+window.artifactRes = [
```
Comment (jshcrowthe) [inline demo/data.json]: Not valid `json`

**Comment by jshcrowthe** [comment]
Other than the point mentioned above this LGTM :smile:

File: demo/data.json (hunk 1 above)
Comment (chadeddington) [inline demo/data.json]: I'm assuming you just wanted me to change the file to be .js instead of .json?

**Comment by jshcrowthe** [comment]
Thanks @chadeddington !

**Pull request merged**